libp2p/__init__.py

```diff
@@ -50,8 +48,8 @@ def generate_peer_id_from(key_pair: KeyPair) -> ID:
 def initialize_default_kademlia_router(
     ksize: int = 20, alpha: int = 3, id_opt: ID = None, storage: IStorage = None
 ) -> KadmeliaPeerRouter:
-    """
-    initialize kadmelia router when no kademlia router is passed in
+    """initialize kadmelia router when no kademlia router is passed in.
```

Comment:
I really dislike this format for docstrings, placing the first line on the same line as the triple quotes.

It appears that PEP257 allows for either via the `--pre-summary-newline` CLI option.  Thoughts on updating this to use that format?

> The summary line may be on the same line as the opening quotes or on the next line.

Reply:
While this is mostly a matter of taste, I agree with the point because the previous format already included a pre-summary newline and I didn't come here to flip everything around completely :smile: Added the pre-summary newline in bafdd85 and updated the `make format` docformatter command

Reply:
Updated the lintroll command and tox CI commands with a doc format check in 3439a2c :+1: